src/views/CustomToken.vue

```diff
@@ -305,6 +328,14 @@ export default {
       this.chainDropdownOpen = false
       this.resetFields()
       this.fetchToken()
+    },
+    isNumber(evt) {
+      const keysAllowed = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
+      const keyPressed = evt.key
+      // The Decimals field can only have 2 digits (0 - 99) and must be a number
+      if (!keysAllowed.includes(keyPressed) || this.decimals.length == 2) {
+        evt.preventDefault()
```

Comment:
would be nice if we could move this to a util file so we could use it in other places

Reply:
@Jennievon It would be, but this function implementation is specific to the fix here. It only validates for whole numbers and up to 2 digits. 

We could refactor this to better suite a general implementation but it would take more time looking into all possible use cases

Reply:
okay then 👍